From 72572a68d306712c040ca741f0655b190326a509 Mon Sep 17 00:00:00 2001
From: Jeremy Tan <jtanx@outlook.com>
Date: Sat, 9 Sep 2017 08:03:57 +0800
Subject: [PATCH] startnoui.c: Include header that defines FindOrMakeEncoding

Failure to include this was causing an implicit pointer to int conversion
which results in an invalid pointer address on 64-bit builds.

Fixes #3145.

diff --git a/fontforgeexe/startnoui.c b/fontforgeexe/startnoui.c
index ab8388539..7a188b7a2 100644
--- a/fontforgeexe/startnoui.c
+++ b/fontforgeexe/startnoui.c
@@ -27,6 +27,7 @@
 #include "fontforgevw.h"
 #include "scripting.h"
 #include "start.h"
+#include "encoding.h"
 
 #ifndef _NO_LIBUNICODENAMES
 #include <libunicodenames.h>	/* need to open a database when we start */
